lib/plugin/remark-mdx.js

```diff
@@ -11,7 +11,8 @@
  */
 
 import syntaxMdxjs from 'micromark-extension-mdxjs'
-import {fromMarkdown, toMarkdown} from 'mdast-util-mdx'
+import pkg from 'mdast-util-mdx';
+const {fromMarkdown, toMarkdown} = pkg;
```

Comment:
```suggestion
import mdx from 'mdast-util-mdx'

const {fromMarkdown, toMarkdown} = mdx
```

Reply:
Yes, I understand. I saw that the #35 was also regarding the same topic. Hopefully the Next.js/webpack team will fix this soon